android/AndroidManifest.xml

```diff
@@ -34,6 +34,7 @@
   <uses-permission android:name="android.permission.CHANGE_WIFI_STATE"/>
   <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
   <uses-permission android:name="android.permission.WAKE_LOCK"/>
+  <uses-permission android:name="android.permission.ACCESS_GPS"/>
```

Comment:
Точно не работает? Сейчас его даже нет в документации https://developer.android.com/reference/android/Manifest.permission.html <https://developer.android.com/reference/android/Manifest.permission.html> и это вроде как устаревшие названия для 
*FINE_LOCATION и *COARSE_LOCATION.

> On Aug 12, 2016, at 3:49 PM, Vladimir Byko-Ianko <notifications@github.com> wrote:
> 
> In android/AndroidManifest.xml <https://github.com/mapsme/omim/pull/4034#discussion_r74584618>:
> 
> > @@ -34,6 +34,7 @@
> >    <uses-permission android:name="android.permission.CHANGE_WIFI_STATE"/>
> >    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
> >    <uses-permission android:name="android.permission.WAKE_LOCK"/>
> > +  <uses-permission android:name="android.permission.ACCESS_GPS"/>
> Без него не работает receiver на PROVIDERS_CHANGED
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/mapsme/omim/pull/4034/files/8c4b1ef3a012bcff35df12e28dea4d2f13f1f24d#r74584618>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ACKTRI7rHdpiYpOY0EdXoqjKXxn9kQn2ks5qfGvYgaJpZM4JiNt2>.
>